### PR TITLE
Create USACO open20-cereal.mdx

### PR DESCRIPTION
**Pull request by Aadit-Ambadkar**

### CODE REVIEW

File: solutions/USACO open20-cereal.mdx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                else {
+                    boxes[match[ocow][1]].add(ocow);
+                }
+            }
+
+        }
+    }
+}
+
+```
```
Comment (HarryWangATX) [inline solutions/USACO open20-cereal.mdx:85]: did not close.

File: solutions/USACO open20-cereal.mdx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+source: USACO Silver 2020 Open
+title: Cereal
+author: Aadit Ambadkar
+---
+
+[Official Analysis](http://usaco.org/current/data/sol_cereal_silver_open20.html)
+
+<LanguageSection>
+<JavaSection>
+## Java Recursive Implementation
```
Comment (HarryWangATX) [inline solutions/USACO open20-cereal.mdx:12]: Make this a solution explanation, bring outside and rename heading.

**Review by HarryWangATX — CHANGES_REQUESTED** [review]

**Comment by HarryWangATX** [comment]
also where is this problem linked on guide???

**Comment by bqi343** [comment]
Please follow *all* of the instructions [here](https://usaco.guide/general/contributing?lang=cpp#adding-a-solution).

**Comment by dongliu0426** [comment]
also please use tabs instead of spaces

**Comment by bqi343** [comment]
Closing, please make a new PR when requested changes have been made.

**Pull request closed without merging**